stdlib/asm/sys.masm

```diff
@@ -4,27 +4,23 @@
 #! function at the end will reduce the size of the public inputs that are shared with the verifier.
 #! Input: Stack with 16 or more elements.
 #! Output: Stack with only the original top 16 elements.
-export.truncate_stack.4
-    loc_storew.0
-    dropw
-    loc_storew.1
-    dropw
-    loc_storew.2
-    dropw
-    loc_storew.3
-    dropw
-    sdepth
-    neq.16
+#! Cycles: 17 + 11 * overflow_words, where `overflow_words` is the number of words needed to drop.
```

Comment:
nit: I'd add a blank line after line 4 (before "Inputs") and another plank line above line 7 (before "Cycles").

Reply:
@Fumuran - could you address this comment as well?